mon-pix/app/components/authentication/login-or-register-oidc.js

```diff
@@ -0,0 +1,102 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import get from 'lodash/get';
+import IdentityProviders from 'mon-pix/identity-providers';
+import isEmailValid from '../../utils/email-validator';
+import isEmpty from 'lodash/isEmpty';
+
+const ERROR_INPUT_MESSAGE_MAP = {
+  termsOfServiceNotSelected: 'pages.login-or-register-oidc.error.error-message',
+  unknownError: 'common.error',
+  expiredAuthenticationKey: 'pages.login-or-register-oidc.error.expired-authentication-key',
+  invalidEmail: 'pages.login-or-register-oidc.error.invalid-email',
+};
+
+export default class LoginOrRegisterOidcComponent extends Component {
+  @service url;
+  @service intl;
+  @service session;
+  @service currentDomain;
+
+  @tracked isTermsOfServiceValidated = false;
+  @tracked isAuthenticationKeyExpired = false;
+  @tracked errorMessage = null;
+  @tracked email = '';
+  @tracked password = '';
+  @tracked emailValidationMessage = null;
+
+  get identityProviderOrganizationName() {
+    return IdentityProviders[this.args.identityProviderSlug].organizationName;
+  }
+
+  get homeUrl() {
+    return this.url.homeUrl;
+  }
+
+  get cguUrl() {
+    const currentLanguage = this.intl.t('current-lang');
+    if (currentLanguage === 'en') {
+      return 'https://pix.org/en-gb/terms-and-conditions';
+    }
+    return `https://pix.${this.currentDomain.getExtension()}/conditions-generales-d-utilisation`;
+  }
+
+  get dataProtectionPolicyUrl() {
+    const currentLanguage = this.intl.t('current-lang');
+    if (currentLanguage === 'en') {
```

Comment:
chipotage: vu que c'est fait a 2 reprises on pourrait faire un `get currentLanguage()`
```suggestion
    if (this.currentLanguage === 'en') {
```

Reply:
done